src/SDKs/BatchAI/Management.BatchAI/node_modules/nodejs/1/getmodule.js

```diff
@@ -0,0 +1,4 @@
+var myMoule = require('./singleobject').Hello("dsf");
+
+/* myMoule.setName("lihao__");
```

Comment:
@lliimsft are these files included by mistake?
If not, can you explain what is the purpose of this file for .NET SDK?